examples/ros/src/smarts_ros/scripts/ros_driver.py

```diff
@@ -640,9 +647,13 @@ def run_forever(self):
 
                 if self._target_freq:
                     if rate.remaining().to_sec() <= 0.0:
-                        rospy.logwarn(
-                            f"SMARTS unable to maintain requested target_freq of {self._target_freq} Hz."
-                        )
+                        msg = f"SMARTS unable to maintain requested target_freq of {self._target_freq} Hz."
```

Comment:
I think it might be useful to log the timestamp if possible.

Reply:
Actually, rospy log messages always include the (ros) timestamp as part of the message.  It's those numbers on the left prior to the message.